source/grid/tria.cc

```diff
@@ -10397,11 +10397,11 @@ Triangulation<dim, spacedim>::get_boundary_ids() const
   else
     {
       std::set<types::boundary_id> b_ids;
-      active_cell_iterator         cell = begin_active();
-      for (; cell != end(); ++cell)
-        for (const unsigned int face : GeometryInfo<dim>::face_indices())
-          if (cell->at_boundary(face))
-            b_ids.insert(cell->face(face)->boundary_id());
+      for (auto cell : cell_iterators())
```

Comment:
This needed to be `active_cell_iterators()` I suppose. We all missed it.

Reply:
Yes you are right! Thanks for noticing!